scripts/mssql_discover.py

```diff
@@ -0,0 +1,70 @@
+import sqlalchemy
+
+MASTER_MSSQL_URL = ""
```

Comment:
This is intentionally blank, so that anyone using this script can add in creds before runtime